response_operations_ui/templates/ce-seft-instrument.html

```diff
@@ -6,52 +6,81 @@
+                                },
+                                "accept": ".xlsx, .xls",
+                                "label": {
+                                    "classes": "ons-u-vh",
+                                    "text": "Choose file"
+                                }
+                            })
+                        -}}
+                    </div>
                 </div>
-            </div>
-            <div>
-                {{
-                    onsButton({
-                        "text": "Upload",
-                        "id": "btn-load-ci",
-                        "variants": ['secondary', 'small', 'disabled'],
-                        "name": "load-ci",
-                        "submitType": "timer"
-                    })
-                }}
-            </div>
-        </form>
+                <div class="ons-u-mb-l">
+                    {% if collection_instruments %}
+                        {{
+                            onsButton({
+                                "text": "Upload",
+                                "id": "btn-load-ci",
+                                "classes": "ons-u-mt-m",
+                                "variants": ['primary', 'small'],
+                                "name": "load-ci",
+                                "submitType": "timer"
+                            })
+                        }}
+                    {% elif not collection_instruments %} 
+                        {{
+                            onsButton({
+                                "text": "Upload",
+                                "id": "btn-load-ci",
+                                "classes": "ons-u-mt-l",
+                                "variants": ['primary'],
+                                "name": "load-ci",
+                                "submitType": "timer"
+                            })
+                        }}
+                        {{
+                            onsButton({
+                                "text": "Cancel",
+                                "id": "btn-load-ci",
+                                "classes": "ons-u-mt-l",
+                                "variants": ['secondary'],
+                                "name": "load-ci-cancel",
+                                "url": url_for('collection_exercise_bp.get_view_sample_ci', short_name=survey.shortName, period=ce.exerciseRef),
+                                "submitType": "timer",
+                                "noIcon": "true"
+                            })
+                        }} 
+                    {% endif %}
+                </div>
+            </form>
+        {% endif %}
     </div>
-    <div class="ons-u-mb-l ons-col-6@m">
-        {% if collection_instruments %}
-            <label class="ons-label ons-u-fs-m">{{ collection_instruments | length }} Collection instruments</label>
+    {% if collection_instruments or (not surveyEditPermission) %}
```

Comment:
This shows the uploaded seft instruments table, if the user has permissions it only shows if there is one uploaded and if the user doesn't have permissions it shows it regardless otherwise the page would be empty